### PR TITLE
pressure management on luos engine

### PR DESCRIPTION
**Pull request by alexgorlier**
# PR Checklist 

### _1- PR Comments_
    *If needed, to be filled by developer*  

<br><hr/>
*WARNING : Do not edit the checklist below.*  
<hr/><br>

### _2- Developer section_

- [x] [Documentation] is up to date with new feature
- [x] [Tests] are *passed OK* (Non regression, new features & bug fixes)
- [x] [Code Quality] Remind to check if :
    * Each function has a header (description, inputs, outputs) 
    * Code is commented (particularly in hard to understand areas)
    * There are no new warnings that can be corrected
    * Commits policy is respected (constitancy commits, clear commits comments)
<hr/><br>

### _3- QA section_

- [x] [Review] tests for new features have been *reviewed*
- [x] [Changelog] is up-to-date with expected tags  
    🆕 Feature: [Feature] Description...  
    🆕 Added: [Feature] Description...  
    🆕 Changed: [Feature] Description...  
    🛠️ Fix: [Feature] Description...

### CODE REVIEW

File: engine/OD/od_pressure.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+static inline pressure_t PressureOD_PressureFrom_Pa(float Pa)
+{
+    return Pa;
+}
+
+// Bars
+static inline float PressureOD_PressureTo_Bars(pressure_t self)
+{
+    return self / 10000.0f;
```
Comment (nicolas-rabault) [inline engine/OD/od_pressure.h:40]: Watch out, PA to Bars is `/100000`.

File: engine/core/inc/luos_list.h [hunk 2]
```diff
@@ -92,7 +93,11 @@ typedef enum
     PARAMETERS, // depend on the service, can be : servo_parameters_t, imu_report_t, motor_mode_t
     ERROR_CMD,
 
+    // Retrocompatibility zone
     TEXT, // ASCII string
+    PRESSURE, // pressure_t (Pa)
+
+
```
Comment (nicolas-rabault) [inline engine/core/inc/luos_list.h:100]: I think you should remove those useless spaces.

File: tool_services/gate/TinyJSON/convert.c [hunk 3]
```diff
@@ -1129,6 +1133,9 @@ const char *Convert_StringFromType(luos_type_t type)
         case PIPE_TYPE:
             return "Pipe";
             break;
+        /*case PRESSURE_TYPE:
+            return "Pressure";
+            break;*/
```
Comment (nicolas-rabault) [inline tool_services/gate/TinyJSON/convert.c:1138]: I don't think you intentionaly add this...

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]
Thank you so much for this PR!
Sound fine, you made a conversion mistake I think, also there is some things you should remove.

**Comment by nicolas-rabault** [comment]
This PR is related to https://github.com/Luos-io/Pyluos/pull/150

File: engine/OD/od_pressure.h (hunk 1 above)
Comment (alexgorlier) [reply engine/OD/od_pressure.h]: Indeed it is my mistake

File: engine/core/inc/luos_list.h (hunk 2 above)
Comment (alexgorlier) [reply engine/core/inc/luos_list.h]: Yes I will

File: tool_services/gate/TinyJSON/convert.c (hunk 3 above)
Comment (alexgorlier) [reply tool_services/gate/TinyJSON/convert.c]: I forgot to save my last updates

**Review by nicolas-rabault — APPROVED** [review]
Sounds good,
An example using this service type could be added to the repo!

**Review by nicolas-rabault — APPROVED** [review]
Great!

**Pull request merged**